content/1.7/back-office/orders/orders/view-shopping-cart.md

```diff
@@ -0,0 +1,36 @@
+---
+title: View Shopping Cart listing
+weight: 1
+---
+
+# SELL > Orders > Shopping Carts > View
+
+[View Shopping Cart](static/img/orders-shopping-carts-view-shopping-cart.png)
+
+This dashboard is displaying the information about the Shopping Cart initiator - the customer, visitor or guest visitor who generated the Cart in Prestashop. It can be accessed by selecting the Shopping Cart row and clicking on View CTA button on the right of the row.
+
+## The elements of the page UI
+
+### Registered customer in Prestashop
+
+- **Total Cart** - the numeric value, that describes the total amount that single customer has so far.
+- **Customer information** - a section, that describes the customer first name, last name, account registration date, valid orders placed, total spent since registration, and the customer email. The customer name and last name contains a link, that directs to the Customer Account page in Prestashop. Email link is also implemented with a link, once clicked initiates the email composing.
+- **Order information** - if the Order is made, then the link appears with the Order number with the hashtag near. This link is clickable, it redirets to the specific order of the Customer has made.
+- **Made on** - the exact timestamp registered when the Order was made.
```

Comment:
I don't know where to find this field, could you show it to me at the next meeting debrief please ? :)

Reply:
@MatShir sure, it is shown only in completed orders - it shows the date, when order was made
![image](https://user-images.githubusercontent.com/29095235/148397750-2e0b9ede-9b14-465b-af70-5cfcbcd8a6a5.png)